flows/verra_data_v2.py

```diff
@@ -44,6 +51,11 @@ def fetch_verra_data_v2_task():
     lst_sn = list(df_bridged_mco2["Serial Number"])
     df.loc[df["Serial Number"].isin(lst_sn), "Moss"] = True
 
+    # Country info
+    df["Country code"] = [
```

Comment:
Should this be capitalized like "Country Code" for consistency with other field names?

Reply:
Ho, I forgot to address that before merging. In practice, the column is renamed by the auto_rename_columns call I do at the end of the function so the dataframe column is ultimately good. But yeah, I will modify it in the next PR. This makes the code ugly.